<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BugReports</class>
 <widget class="QWidget" name="BugReports">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>926</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="Background" native="true">
     <property name="styleSheet">
      <string notr="true">#Background
{
background-color: #F7F8FA;
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="MainWidget" native="true">
        <property name="styleSheet">
         <string notr="true">#MainWidget
{
background-image: url(:/gui/ui/asset/bg_dark-stripes.png);
}</string>
        </property>
        <widget class="QPushButton" name="btnAttachScreenshot">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>392</y>
           <width>228</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #DA0B82;
font: 16px/22px;
font-weight: normal;
font-style: normal;
font-size: normal;
font-family: 'Lato';
background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Click here to attach a screenshot</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="KelGuiPushButton" name="btnReturn">
         <property name="geometry">
          <rect>
           <x>33</x>
           <y>61</y>
           <width>30</width>
           <height>30</height>
          </rect>
         </property>
         <property name="style">
          <enum>KelGuiPushButton::Custom</enum>
         </property>
         <property name="custom">
          <url>
           <string>qrc:/gui/ui/asset/ic_close.png</string>
          </url>
         </property>
         <property name="customHover">
          <url>
           <string>qrc:/gui/ui/asset/ic_close.png</string>
          </url>
         </property>
         <property name="customPushed">
          <url>
           <string>qrc:/gui/ui/asset/ic_close.png</string>
          </url>
         </property>
        </widget>
        <widget class="QLabel" name="labelLetterAmount">
         <property name="geometry">
          <rect>
           <x>297</x>
           <y>333</y>
           <width>96</width>
           <height>19</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #A4A3C0;
font: 16px/25px;
font-weight: normal;
font-style: normal;
font-size: normal;
font-family: 'Lato';
background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="text">
          <string>0/200</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="QListView" name="listView">
         <property name="geometry">
          <rect>
           <x>8</x>
           <y>121</y>
           <width>392</width>
           <height>521</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0,0,0,0);</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelBugReports">
         <property name="geometry">
          <rect>
           <x>73</x>
           <y>62</y>
           <width>282</width>
           <height>28</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #04004E;
font: 24px/29px;
font-weight: normal;
font-style: normal;
font-size: bold;
font-family: 'Lato';
background-color: rgba(0, 0, 0, 0);</string>
         </property>
         <property name="text">
          <string>Bug reports</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
        <widget class="KelGuiPlainTextEdit" name="editReport">
         <property name="geometry">
          <rect>
           <x>19</x>
           <y>133</y>
           <width>390</width>
           <height>200</height>
          </rect>
         </property>
         <property name="placeholderText">
          <string>Please describe the details of problem you faced. What actions did you take and what happened.</string>
         </property>
        </widget>
        <widget class="KelGuiPushButton" name="btnSendReport">
         <property name="geometry">
          <rect>
           <x>27</x>
           <y>441</y>
           <width>376</width>
           <height>82</height>
          </rect>
         </property>
         <property name="text">
          <string>send report</string>
         </property>
        </widget>
        <widget class="QLabel" name="labelLoading">
         <property name="geometry">
          <rect>
           <x>171</x>
           <y>229</y>
           <width>86</width>
           <height>86</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>86</width>
           <height>86</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>86</width>
           <height>86</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">image: url(:/gui/ui/asset/Spinner.gif);</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QWidget" name="layoutWidget">
         <property name="geometry">
          <rect>
           <x>78</x>
           <y>20</y>
           <width>301</width>
           <height>24</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>TestMode:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTestList">
            <property name="text">
             <string>List</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTestWrite">
            <property name="text">
             <string>Write</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioTestLoading">
            <property name="text">
             <string>Loading</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KelGuiPushButton</class>
   <extends>QPushButton</extends>
   <header>kelguipushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KelGuiPlainTextEdit</class>
   <extends>QWidget</extends>
   <header>kelguiplaintextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
